godot-macros/src/class/data_models/inherent_impl.rs

```diff
@@ -84,6 +86,8 @@ pub fn transform_inherent_impl(
     let (funcs, signals) = process_godot_fns(&class_name, &mut impl_block, meta.secondary)?;
     let consts = process_godot_constants(&mut impl_block)?;
 
+    let func_export_name_constants = make_function_registered_name_constants(&funcs, &class_name);
```

Comment:
```suggestion
    let func_name_constants = make_function_registered_name_constants(&funcs, &class_name);
```
"export" has a very specific meaning; using it here is probably confusing.

Reply:
it was also not consistent with the function name. I'd propose standardizing on "exported" function name, at least that's what it's called in other places in the code.